stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -0,0 +1,181 @@
+#ifndef STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_LPMF_HPP
+#define STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_LPMF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <boost/random/bernoulli_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+#include <cmath>
+#include <string>
+
+using Eigen::Dynamic;
+using Eigen::Matrix;
+
+namespace stan
+{
+namespace math
+{
+
+/**
+     * Returns the log PMF of the Generalized Linear Model (GLM)
+     * with Bernoulli distribution and logit link function.
+     * If containers are supplied, returns the log sum of the probabilities.
+     * @tparam T_n type of integer parameter
+     * @tparam T_x type of the data matrix (design matrix);
+     * this should be an Eigen::Matrix type
+     * @tparam T_beta type of the weight vector
+     * @tparam T_alpha type of the bias vector
```

Comment:
This is an intercept or location term, not a "bias" ("bias" has a precise meaning in stats, and this isn't it, despite a lot of people calling it that).

`T_alpha` should only be a scalar.  I don't even know what making it a vector would mean.

Reply:
OK. I guess you could make it a vector if you have a reason to believe that say, the first half of your dataset has a different intercept from the second half. (alpha = [alpha1, ..., alpha1, alpha2, ..., alpha2]) I wanted to write the function in as much generality as I could imagine people using it. I've changed the description to simply be intercept though.

Reply:
I've changed this to just be a scalar now.